gempython/tools/optohybrid_user_functions_uhal.py

```diff
@@ -594,6 +594,10 @@ def configureScanModule(device, gtx, mode, vfat, channel=0,
         pass
 
     writeRegisterList(device,regList)
+    
+    regList = dict.fromkeys(["GEM_AMC.OH.OH%d.COUNTERS.CRC.INCORRECT.VFAT%d.Reset"%(gtx,i) for i in range(24)],1)
```

Comment:
for the moment for readability. But I can do it after the mods are properly tested